frontend/src/components/ex-card.js

```diff
@@ -0,0 +1,53 @@
+import {html, PolymerElement} from '@polymer/polymer/polymer-element.js';
+
+/**
+ * `ex-card`
+ *
+ * ExCard example card component.
+ *
+ * @customElement
+ * @polymer
+ */
+class ExCard extends PolymerElement {
+
+    static get template() {
+        return html`
+            <style include="common-styles">
+                :host {
+                    display: flex;
+                    flex-direction: column;
+                    box-shadow: var(--lumo-box-shadow-xs);
+                    border-radius: var(--lumo-border-radius-m);
+                    border: 1px solid var(--lumo-shade-10pct);
```

Comment:
You should incorporate this border in to the box-shadow instead, it’ll create a better visual end result.

`box-shadow: 0 0 0 1px var(--lumo-contrast-5pct), var(--lumo-box-shadow-xs);`

Reply:
Also, I suggest you apply a background for the card, so that it looks better with the dark palette:

`background: var(--lumo-base-color) linear-gradient(var(--lumo-tint-5pct), var(--lumo-tint-5pct));`

Reply:
> `box-shadow: 0 0 0 1px var(--lumo-contrast-5pct), var(--lumo-box-shadow-xs);`

Thanks for the tip, I'll update it to use this! This crossed my mind initially but I wasn't sure how to do it as I haven't worked that much with box-shadows and didn't want to spend too much time on it.

Also good idea about making sure this works nicely with dark theme too. I'll test it and make some small style updates.

Reply:
Done. Also updated some other colors for dark theme (mainly text that was dark on dark).